x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -4,64 +4,107 @@
 
   describe('rollup job', function () {
-    this.tags('smoke');
-    before(async () => {
-      // init data
-      await esArchiver.loadIfNeeded('logstash_functional');
-      await esArchiver.load('canvas/default');
+    const rollupJobName = 'rollup-to-be-' + Math.floor(Math.random() * 10000);
+    const indexName = 'rollup-to-be';
+    const now = new Date();
+    const pastDates = [
+      datemath.parse('now-1d', { forceNow: now }),
+      datemath.parse('now-2d', { forceNow: now }),
+      datemath.parse('now-3d', { forceNow: now }),
+    ];
+
+    async function loadDates(dates, prepend = 'to-be-rolled-up') {
+      for (const day of dates) {
+        await es.index({
+          index: `${prepend}-${day.format('MM-DD-YYYY')}`,
+          body: {
+            '@timestamp': day.toISOString(),
+            foo_metric: 1
+          }
+        });
+      }
+    }
+
+    it('create new rollup job', async () => {
+      const indexPattern = 'to-be*';
+      const interval = '1000ms';
+
+      await log.debug('make sure all dates have the same concept of "now"');
+      await log.debug(pastDates);
+      await loadDates(pastDates);
+
+      await PageObjects.common.navigateToApp('rollupJob');
+      await PageObjects.rollup.createNewRollUpJob(rollupJobName, indexPattern, indexName,
+        interval, ' ', true, { time: '*/10 * * * * ?', cron: true });
+
+      await PageObjects.common.navigateToApp('indexManagement');
+
+      await PageObjects.indexManagement.toggleRollupIndices();
+
+      await log.debug('If the index exists then the rollup job has successfully' +
+        'been created and is waiting to or has triggered.');
+
+      //Stop the running rollup job.
+      await es.transport.request({
+        path: `/_rollup/job/${rollupJobName}/_stop?wait_for_completion=true`,
+        method: 'POST',
+      });
+
+
+    });
+
+    it('create hybrid index pattern', async () => {
+
+      await log.debug('Create data for the rollup job to recognize.');
+
+      await pastDates.map(async (day) => {
+        await es.index(await getRolledUpData(rollupJobName, day));
+      });
       await PageObjects.common.navigateToApp('rollupJob');
+
+      //Index live data to be used in the test.
+
+      await log.debug('Add data for 1,2 and 3 days into the future.');
+      const futureDates = [
+        datemath.parse('now', { forceNow: now }),
+        datemath.parse('now+1d', { forceNow: now }),
+        datemath.parse('now+2d', { forceNow: now }),
+        datemath.parse('now+3d', { forceNow: now }),
+      ];
+
+      await loadDates(futureDates, 'live-data');
+
+      await PageObjects.common.navigateToApp('settings');
+      await PageObjects.settings.createIndexPattern('live*,' + indexName, '@timestamp', false);
+
+      await PageObjects.common.navigateToApp('discover');
+      const hits = await PageObjects.discover.getHitCount();
```

Comment:
I haven't run this yet but I'm surprised this works since Discover usually defaults to `Last 15 minutes` and that seems like it would only have 1 doc at most in it?  Aren't the other docs spread across 6 days?

Reply:
This has shown all hits. Has the default possibly changed? The test does work.

Reply:
oh, I think I recall now.  The behavior is not great on Discover with hybrid index patterns.  It treats it as though it's not a time-based index because the live data has a timefield like @timestamp and the rollup data has some different variations of that.  So discover just shows all the docs and no timepicker.